Tests/AppcuesKitTests/Traits/TraitComposerTests.swift

```diff
@@ -459,6 +458,48 @@ extension TraitComposerTests {
     }
 
     class TestTrait: StepDecoratingTrait, ContainerCreatingTrait, ContainerDecoratingTrait, WrapperCreatingTrait, BackdropDecoratingTrait {
+        struct Config: Decodable {
+            let groupID: String?
+            let stepDecoratingExpectation: DecodableExpectation?
+            let containerCreatingExpectation: DecodableExpectation?
+            let containerDecoratingExpectation: DecodableExpectation?
+            let wrapperCreatingExpectation: DecodableExpectation?
+            let backdropDecoratingExpectation: DecodableExpectation?
+
+            init(groupID: String? = nil,
+                 stepDecoratingExpectation: XCTestExpectation? = nil,
+                 containerCreatingExpectation: XCTestExpectation? = nil,
+                 containerDecoratingExpectation: XCTestExpectation? = nil,
+                 wrapperCreatingExpectation: XCTestExpectation? = nil,
+                 backdropDecoratingExpectation: XCTestExpectation?) {
+                self.groupID = groupID
+                if let stepDecoratingExpectation = stepDecoratingExpectation {
+                    self.stepDecoratingExpectation = DecodableExpectation(expectation: stepDecoratingExpectation)
+                } else {
+                    self.stepDecoratingExpectation = nil
+                }
```

Comment:
Nothing wrong with this (verbosity is fine in tests), but if you want to make it a 1-liner you could make the init accept an optional expectation and then return `nil` if that's what you get.

Reply:
right, thank you - happy to compress this.